src/components/Auth/Auth.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
```

Comment:
Group material-ui/core/... in one string. 
import { Tab, Tabs} from "@material-ui/core/Tab";

Reply:
fixed